lib/internal/modules/package_json_reader.js

```diff
@@ -87,52 +107,93 @@ function read(jsonPath, { base, specifier, isESM } = kEmptyObject) {
     specifier == null ? undefined : `${specifier}`,
   );
 
-  return deserializePackageJSON(jsonPath, parsed);
+  const result = deserializePackageJSON(jsonPath, parsed);
+
+  return {
+    ...result.data,
```

Comment:
Null prototyping?

Reply:
I don't understand—there is `__proto__: null` right above your comment

Reply:
IIUC you are creating a new object, show it should have it's own null prototype, but I might be wrong

Reply:
Something is wrong with the display. There is actually a `__proto__: null` here